hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -1379,6 +1313,8 @@ static bool vlapic_find_deliverable_intr(const struct acrn_vlapic *vlapic, uint3
  * @return None
  *
  * @pre vlapic != NULL
+ * @pre vlapic->isrvec_stk_top will small than ISRVEC_STK_SIZE
```

Comment:
will be smaller than

Reply:
BTW, this kind of @pre is not compilable. It should be a simple expression. Not text.

Reply:
Thanks. Will update this.